resources/lib/vrtplayer/metadatacollector.py

```diff
@@ -50,6 +52,22 @@ def get_plot(soup):
         return description
 
     @staticmethod
+    def get_broadcast_date(soup):
+        broadcast_date_tag = soup.find(class_="content__broadcastdate")
+        
+        if broadcast_date_tag is None:
```

Comment:
Will do - I was following the [*fail fast*][1] principle, as explained in an [article by Martin Fowler][2] (quite an interesting read, by the way - would recommend it). I'm following this principle for years, it's a great help in my experience.

I'll adapt this weekend.

[1]: https://en.wikipedia.org/wiki/Fail-fast
[2]: https://martinfowler.com/ieeeSoftware/failFast.pdf